packages/shared/src/components/cards/squad/UnfeaturedSquadGrid.tsx

```diff
@@ -0,0 +1,79 @@
+import React, { ReactElement } from 'react';
+import classNames from 'classnames';
+import { Card, CardLink } from '../Card';
+import { ButtonVariant } from '../../buttons/Button';
+import {
+  Typography,
+  TypographyColor,
+  TypographyTag,
+  TypographyType,
+} from '../../typography/Typography';
+import { largeNumberFormat } from '../../../lib';
+import { Separator } from '../common';
+import { SquadImage } from './common/SquadImage';
+import { SquadJoinButtonWrapper } from './common/SquadJoinButton';
+import { UnFeaturedSquadCardProps } from './common/types';
+
+export const UnfeaturedSquadGrid = ({
+  source,
+  title,
+  subtitle,
+  icon,
+  action,
+  description,
+}: UnFeaturedSquadCardProps): ReactElement => {
+  return (
+    <Card className={classNames('overflow-hidden border-0 p-4')}>
+      <CardLink
+        href={source.permalink}
+        rel="noopener"
+        title={source.description}
+      />
+      <div className="mb-3 flex items-center justify-between">
+        <SquadImage
+          image={source?.image}
+          icon={icon}
+          title={title}
+          size="size-16"
+        />
+        <SquadJoinButtonWrapper
+          action={action}
+          source={source}
+          variant={ButtonVariant.Float}
+          className={{
+            squadJoinButton: '!btn-tertiaryFloat',
+          }}
+        />
+      </div>
+      <Typography
+        tag={TypographyTag.H1}
+        type={TypographyType.Body}
+        bold
+        truncate
+      >
+        {title}
+      </Typography>
+      <Typography
+        tag={TypographyTag.P}
+        type={TypographyType.Callout}
+        color={TypographyColor.Tertiary}
+        className="multi-truncate line-clamp-2"
+      >
+        {source?.description || description}
+      </Typography>
+
+      {subtitle && (
+        <Typography
+          color={TypographyColor.Tertiary}
+          type={TypographyType.Footnote}
+          className="mt-2"
+        >
+          {subtitle} <Separator />
+          <strong data-testid="squad-members-count">
+            {largeNumberFormat(source.membersCount)}
```

Comment:
I think we are missing the `members` label on this one. Design team may have missed updating the Sketch but even on Cards it must have the same label.

![Screenshot 2024-08-23 at 9 32 14 PM](https://github.com/user-attachments/assets/76ae9dc0-106d-43a7-bfbb-e289633ef522)